pkgs/games/runescape-launcher/wrapper.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildFHSUserEnv, runescape-launcher-runtime }:
+
+buildFHSUserEnv rec {
+  name = "runescape-launcher-wrapper-${runescape-launcher-runtime.version}";
+
+  runScript = "${runescape-launcher-runtime}/bin/runescape-launcher";
+
```

Comment:
@fusion809 Could you add `profile = "export LD_DEBUG=libs";` here and show us output of `runescape-launcher`? I didn't want to do this initially because it's difficult to analyze but it could give us hints on what happens.

Reply:
What add it to the file or run it at the command-line? I ran this package in NixOS itself and it ran fine.

Reply:
Gave it a go, adding it to the wrapper.nix file, which is what I assume you mean, and I got the error:

```bash
/nix/store/iwxs4nwk80arwkvqdjqcy74r34wcg3wl-chroot-user:45:in `block in make_fcall': Invalid argument (Errno::EINVAL)
        from /nix/store/iwxs4nwk80arwkvqdjqcy74r34wcg3wl-chroot-user:115:in `<main>'
```

Reply:
Gave it a go, adding it to the wrapper.nix file, which is what I assume you mean and it crashes and before it does it's got that strange font rendering issue:

![screenshot_20171117_170924](https://user-images.githubusercontent.com/4717341/32935159-90e3bc32-cbba-11e7-8b60-bd5536d5fc95.png)

Here is the output: https://gist.github.com/fec0b781d1b077af4940e475fa7868d3.

Reply:
Any idea where might `/var/bamboo-home/bamboo-home-05/xml-data/build-dir/NXT-BUIL2082-BLR/devplatform/3rdPartyLibs/cpp/wxWidgets/lib/linux/Release/tls/x86_64` come from?

Reply:
Haven't the foggiest if you're asking me. I'm not a panda so I have no need for bamboo :P.

Reply:
Ah, it may be paths from Runescape binaries then.